Gemfile.lock

```diff
@@ -0,0 +1,710 @@
+      slop (~> 3.4)
+    pry-byebug (3.3.0)
+      byebug (~> 8.0)
+      pry (~> 0.10)
+    pry-remote (0.1.8)
+      pry (~> 0.9)
+      slop (~> 3.0)
+    pry-stack_explorer (0.4.9.2)
+      binding_of_caller (>= 0.7)
+      pry (>= 0.9.11)
+    r-train (0.10.4)
+      docker-api (~> 1.26.2)
+      json (~> 1.8)
+      mixlib-shellout (~> 2.1)
+      net-scp (~> 1.2)
+      net-ssh (>= 2.9, < 4.0)
+      winrm (~> 1.6)
+      winrm-fs (~> 0.3)
+    rack (1.6.4)
+    rainbow (2.1.0)
+    rake (11.1.2)
+    rb-fsevent (0.9.7)
+    rb-inotify (0.9.7)
+      ffi (>= 0.5.0)
+    rb-readline (0.5.3)
+    rbvmomi (1.8.2)
+      builder
+      nokogiri (>= 1.4.1)
+      trollop
+    rdoc (4.2.2)
+      json (~> 1.4)
+    retriable (1.4.1)
+    retryable (2.0.3)
+    ridley (4.5.0)
+      addressable
+      buff-config (~> 1.0)
+      buff-extensions (~> 1.0)
+      buff-ignore (~> 1.1)
+      buff-shell_out (~> 0.1)
+      celluloid (~> 0.16.0)
+      celluloid-io (~> 0.16.1)
+      chef-config (>= 12.5.0)
+      erubis
+      faraday (~> 0.9.0)
+      hashie (>= 2.0.2, < 4.0.0)
+      httpclient (~> 2.7)
+      json (>= 1.7.7)
+      mixlib-authentication (>= 1.3.0)
+      retryable (~> 2.0)
+      semverse (~> 1.1)
+      varia_model (~> 0.4.0)
+    rspec (3.4.0)
+      rspec-core (~> 3.4.0)
+      rspec-expectations (~> 3.4.0)
+      rspec-mocks (~> 3.4.0)
+    rspec-core (3.4.4)
+      rspec-support (~> 3.4.0)
+    rspec-expectations (3.4.0)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.4.0)
+    rspec-its (1.2.0)
+      rspec-core (>= 3.0.0)
+      rspec-expectations (>= 3.0.0)
+    rspec-mocks (3.4.1)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.4.0)
+    rspec-support (3.4.1)
+    rspec_junit_formatter (0.2.3)
+      builder (< 4)
+      rspec-core (>= 2, < 4, != 2.12.0)
+    rubocop (0.37.2)
+      parser (>= 2.3.0.4, < 3.0)
+      powerpack (~> 0.1)
+      rainbow (>= 1.99.1, < 3.0)
+      ruby-progressbar (~> 1.7)
+      unicode-display_width (~> 0.3)
+    ruby-prof (0.15.9)
+    ruby-progressbar (1.7.5)
+    rubyntlm (0.6.0)
+    rubyzip (1.2.0)
+    rufus-lru (1.0.5)
+    safe_yaml (1.0.4)
+    sawyer (0.7.0)
+      addressable (>= 2.3.5, < 2.5)
+      faraday (~> 0.8, < 0.10)
+    semverse (1.2.1)
+    serverspec (2.31.0)
+      multi_json
+      rspec (~> 3.0)
+      rspec-its
+      specinfra (~> 2.53)
+    sfl (2.2)
+    shellany (0.0.1)
+    signet (0.7.2)
+      addressable (~> 2.3)
+      faraday (~> 0.9)
+      jwt (~> 1.5)
+      multi_json (~> 1.10)
+    slop (3.6.0)
+    solve (2.0.3)
+      molinillo (~> 0.4.2)
+      semverse (~> 1.1)
+    specinfra (2.55.0)
+      net-scp
+      net-ssh (>= 2.7, < 3.1)
+      net-telnet
+      sfl
+    stuartpreston-azure-sdk-for-ruby (0.7.1)
+      json (~> 1.8)
+      mime-types (~> 2.0)
+      nokogiri (~> 1.5)
+      systemu (~> 2.6)
+      uuid (~> 2.0)
+    syslog-logger (1.6.8)
+    systemu (2.6.5)
+    test-kitchen (1.6.0)
```

Comment:
One issue right now is we can't do 1.7.1 until chef-provisioning and test-kitchen stop having conflicting deps.